CHANGELOG.md

```diff
@@ -85,13 +85,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: Biotech Research Area 2 can now be crossed with a Hypermode Wall Boost
 
+##### Impact Crater
+
+##### Magmoor Caverns
+
+- Added: The Sloped R-Jump in Geothermal Core to get to the door to Plasma Processing (commonly referred to as Eagle Jump) is now in logic.
```

Comment:
This seems wrong, i cant see this in the txt files

Reply:
This was a change made by Schwartz. I rearranged the Changelog entry to fit the areas alphabetically.